Makefile

```diff
@@ -312,9 +312,14 @@ $(ONIGURUMA):
 
 redisearch: $(REDISEARCH_LIBS)
 
+DEBUG.search=$(DEBUG)
+ifeq ($(COV), 1)
+DEBUG.search=0
+endif
+
 $(REDISEARCH_LIBS):
 	@echo Building $@ ...
-	$(SHOW)$(MAKE) -C $(REDISEARCH_DIR) STATIC=1 BINROOT=$(REDISEARCH_BINROOT) CC=$(CC) CXX=$(CXX)
+	$(SHOW)$(MAKE) -C $(REDISEARCH_DIR) COV=$(COV) DEBUG=$(DEBUG.search) STATIC=1 CC=$(CC) CXX=$(CXX) IGNORE_MISSING_DEPS=1
```

Comment:
Looks good, why did you add `IGNORE_MISSING_DEPS` which dependencies might be missing?